airflow/www/templates/airflow/dag.html

```diff
@@ -315,6 +315,19 @@ <h4 class="modal-title" id="dagModalLabel">
     var task_id = '';
     var exection_date = '';
     var subdag_id = '';
+
+    var buttons = {
+      subDag: document.querySelector('#btn_subdag'),
+      task: document.querySelector('#btn_task'),
+      rendered: document.querySelector('#btn_rendered'),
+      instances: document.querySelector('#btn_ti'),
+      log: document.querySelector('#btn_log'),
```

Comment:
Could we do something like `document.querySelectorAll("a[data-base-url]")` to find any a tag with data-base-url attribute?

Reply:
Each button gets an individual link, so it would be difficult.